impeller/compiler/shader_lib/impeller/gaussian.glsl

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GAUSSIAN_GLSL_
+#define GAUSSIAN_GLSL_
+
+#include <impeller/constants.glsl>
+
+float IPGaussian(float x, float sigma) {
```

Comment:
What's the meaning of the `IP` prefix?

Reply:
It's the C style namespacing of shader library functions.